spec/unit/oidc/authorize.spec.ts

```diff
@@ -0,0 +1,183 @@
+/*
+Copyright 2023 The Matrix.org Foundation C.I.C.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import fetchMockJest from "fetch-mock-jest";
```

Comment:
It's by no means universal, but it seems like this is more normally imported as `fetchMock` rather than `fetchMockJest`. Being consistent makes it easier to move code around. YMMV though!

Reply:
Using the automatic import name seems more sensible to me but have renamed to `fetchMock`.
(I wonder if intellisense infers different import names on different IDEs?)

Reply:
I think the convention of using `fetchMock` comes from the [readme for fetch-mock-jest](https://github.com/wheresrhys/fetch-mock-jest#global-fetch), and that's probably because it's basically exporting the same API as fetch-mock? :man_shrugging: